src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -111,6 +114,38 @@ function validateDangerousTag(tag) {
   }
 }
 
+var processStyleName = memoizeStringOnly(function(styleName) {
```

Comment:
There should probably be a comment here noting that the code here is copied in ReactDOMComponent.